.dockerignore

```diff
@@ -0,0 +1,3 @@
+*
+!docker/
+!pkg/docker
```

Comment:
I didn't want to interfere with upstream's `docker` directory, so created a `pkg/docker` directory instead.

Previously it was `deb-package/docker`, and this seemed to work just fine.